WordPress/Sniffs/Theme/NoTitleTagSniff.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @category PHP
+ * @package  PHP_CodeSniffer
+ * @link     https://make.wordpress.org/core/handbook/best-practices/coding-standards/
+ */
+
```

Comment:
Please update the file documentation to follow the (new) pattern as introduced in https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/pull/651